.github/workflows/ci.yml

```diff
@@ -153,6 +153,7 @@ jobs:
 
   build_macos:
     strategy:
+      fail-fast: false
```

Comment:
This might be useful in the future when something breaks only in a certain configuration or with a certain python version, thus I left it here.